libyul/backends/wasm/WordSizeTransform.cpp

```diff
@@ -208,7 +208,8 @@ void WordSizeTransform::run(Dialect const& _inputDialect, Block& _ast, NameDispe
 {
 	// Free the name `or_bool`.
 	NameDisplacer{_nameDispenser, {"or_bool"_yulstring}}(_ast);
-	WordSizeTransform{_inputDialect, _nameDispenser}(_ast);
+	YulString defaultType; // should be i64 at some point.
+	WordSizeTransform{_inputDialect, _nameDispenser, defaultType}(_ast);
```

Comment:
```suggestion
	WordSizeTransform{_inputDialect, _nameDispenser, YulString{}}(_ast);
```
Or does the comment needs to be addressed in this PR? Otherwise add a `/// TODO`?

Reply:
I think it's better to name the argument.

Reply:
The comment does not need to be addressed because the type checks are not implemented yet.

Reply:
ok!